css-brochure-form/styles.css

```diff
@@ -0,0 +1,77 @@
+h2 {
+  margin: auto;
+  text-align: center;
+  color: #ef7772;
+  font-family: 'Brandon Grotesque', sans-serif;
+  letter-spacing: 1.5px;
+}
+
+div {
```

Comment:
Applying this to the `div` element type is dangerous business. `div` elements are extremely popular in HTML, and with this, you will have permanently changed their behavior. You would be better off adding a class to the `div` this styling is for, and then make this rule for that specific class instead.